conf/log4j2.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- All loggers are Async if
+    -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector if is set -->
+<Configuration status="warn" strict="true">
+  <Properties>
+    <Property name="consoleLayoutPattern">%d{ISO8601} [myid:%X{myid}] - %-5p [%t:%C{1}@%L] - %m%n</Property>
```

Comment:
I was trying to make sure that I can generate log4j2.xml configuration compatible with the current log4j.properties